docs/BreakingChanges/056 Support for type forwarding when multi-targeting.md

```diff
@@ -0,0 +1,22 @@
+## 56: Support for type forwarding when multi-targeting
```

Comment:
{
  "Id": "56",
  "Title": "Support for type forwarding when multi-targeting",
  "Details": "A new CodeDOM feature allows a compiler to compile against the targeted version of mscorlib.dll instead of the .NET Framework 4.5 version of mscorlib.dll.",
  "Suggestion": "This change prevents compiler warnings (and compilation failure in cases where warnings are treated as errors) when CodeDOM finds two definitions for types that have been type-forwarded. This change may have unintended side effects only if different versions of reference assemblies are mixed in a single location.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#core",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [ ] Not detectable via API analysis